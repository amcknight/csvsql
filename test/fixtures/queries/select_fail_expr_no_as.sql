SELECT 2 + 3, first_name
FROM test/fixtures/tables/people
