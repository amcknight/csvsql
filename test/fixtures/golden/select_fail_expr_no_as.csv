1:3:
  |
1 | 2 + 3, first_name
  |   ^
unexpected '+'
expecting ',', end of input, or white space
